modules/core_project_factory/variables.tf

```diff
@@ -175,6 +175,12 @@ variable "bucket_labels" {
   default     = {}
 }
 
+variable "bucket_force_destroy" {
+  description = "Enable to detele all objects within the GCS bucket. If you try to delete a bucket that contains objects, Terraform will fail that run. (optional)"
```

Comment:
This description isn't accurate. Please make it clearer that Terraform will force-delete objects in the bucket (*not* fail) if this setting is active.

Reply:
@morgante I've just pushed a different description. Hopefully it's clearer this time. If you still want changes let me know what description you'd like.